src/overlays/actors/ovl_Obj_HsStump/z_obj_hsstump.c

```diff
@@ -23,18 +28,129 @@ const ActorInit Obj_HsStump_InitVars = {
 };
 */
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_Obj_HsStump_0x80BDAA30/ObjHsStump_Init.asm")
+extern Gfx D_060003B8[];
+
+extern CollisionHeader D_060011B0;
+
+extern InitChainEntry D_80BDAF80[];
+/*
+static InitChainEntry sInitChain[] = {
+    ICHAIN_VEC3F_DIV1000(scale, 180, ICHAIN_STOP),
+};
+*/
+
+extern Vec3f D_80BDAF84;
+// static Vec3f iceSmokeAccel = {0.0f, 0.0f, 0.0f};
```

Comment:
So, my understanding is that you can't fully switch to the recompiled overlay in `linker_script/code_script.txt` util the last function matches. (The `.ovl` section won't match because it will be missing some relocation symbols).

Locally I was able to change just the `.data` section, which allowed me to uncomment all these variables.
```diff
diff --git a/linker_scripts/code_script.txt b/linker_scripts/code_script.txt
index a3afc53..ad8047c 100644
--- a/linker_scripts/code_script.txt
+++ b/linker_scripts/code_script.txt
@@ -8239,7 +8239,7 @@ SECTIONS
     ovl_Obj_HsStump : AT(RomLocation)
     {
         build/src/overlays/actors/ovl_Obj_HsStump/z_obj_hsstump.o(.text)
-        build/asm/overlays/ovl_Obj_HsStump_data.o(.data)
+        build/src/overlays/actors/ovl_Obj_HsStump/z_obj_hsstump.o(.data)
         build/src/overlays/actors/ovl_Obj_HsStump/z_obj_hsstump.o(.rodata)
         build/asm/overlays/ovl_Obj_HsStump_rodata.o(.rodata)
     }
```